elftools/dwarf/datatype_cpp.py

```diff
@@ -212,11 +212,11 @@ def get_class_spec_if_member(func_spec, the_func):
 def format_function_param(param_spec, param):
     if param_spec.tag == 'DW_TAG_formal_parameter':
         if 'DW_AT_name' in param.attributes:
-            name = DIE_name(param)
+            DIE_name(param)
```

Comment:
hmm, is this needed at all? AFAICT `DIE_name` doesn't have side effects

Reply:
I've added another commit to remove the whole `if-then-else` block as `name` is used nowhere.